beetsplug/replaygain.py

```diff
@@ -1310,13 +1309,14 @@ def album_requires_gain(self, album):
                 for item in album.items()])
 
     def _store(self, item):
-        try:
-            item.store()
-        except OperationalError:
-            # test_replaygain.py :memory: library can fail with
-            #    `sqlite3.OperationalError: no such table: items`
-            # but the second attempt succeeds
-            item.store()
+        """Store an item to the database.
+             When testing, item.store() sometimes fails non-destructively with
+             sqlite.OperationalError.
+             This method is here to be patched to a retry-once helper function
+             in test_replaygain.py, so that it can still fail appropriately
+             outside of these tests.
+        """
+        item.store()
```

Comment:
Removed the `OperationalError` handler, added an explanation as to why this seemingly useless method is there